Utils/Dataflow/pyDKB/common/_logging.py

```diff
@@ -0,0 +1,286 @@
+"""
+pyDKB.common._logging
+
+A wrapper for standard 'logging' module.
+"""
+
+import logging
+from logging import CRITICAL, FATAL, ERROR, WARNING, WARN, INFO, DEBUG, NOTSET
```

Comment:
@mgolosova 
What is the main difference between "critical" and "fatal" errors (especially in pyDKB)?

Reply:
@anastasiakaida,
I guess [this](https://github.com/python/cpython/blob/2.7/Lib/logging/__init__.py#L124) is the answer ;)